tests/unit/test_variable.py

```diff
@@ -112,6 +117,87 @@ def test_exists_hit_w_alternate_client(self):
         self.assertEqual(req["path"], "/%s" % (self.PATH,))
         self.assertEqual(req["query_params"], {"fields": "name"})
 
+    def test_create_no_data(self):
+        from google.cloud.runtimeconfig.config import Config
+
+        conn = _Connection()
+        client = _Client(project=self.PROJECT, connection=conn)
+        config = Config(name=self.CONFIG_NAME, client=client)
+        variable = config.variable(self.VARIABLE_NAME)
+        with self.assertRaises(TypeError) as ctx:
+            variable.create()
+        self.assertEqual("No text or value set.", str(ctx.exception))
```

Comment:
I don't know what the convention for the rest of the codebase is, but this is a general a code smell I like to avoid via oneoff exception classes.

Reply:
Good point, I did not see any defined and wanted to minimize impact. Would your preference be for finding a suitable one exposed by the core modules, or defining a `RuntimeConfigError` or somesuch?

Reply:
I had a look at other googleapis libraries and did not see a pattern emerge, so I added a new `exceptions` module and an `Error` class. Let me know if this is ok.